src/test/cloudWatchLogs/commands/searchLogGroup.test.ts

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as assert from 'assert'
+import {
+    SearchLogGroupWizard,
+    createLogGroupPrompter,
+    createFilterpatternPrompter,
+} from '../../../cloudWatchLogs/commands/searchLogGroup'
+import { AWSTreeNodeBase } from '../../../shared/treeview/nodes/awsTreeNodeBase'
+import { createQuickPickTester } from '../../shared/ui/testUtils'
+import { exposeEmitters, ExposeEmitters } from '../../../../src/test/shared/vscode/testUtils'
+import { InputBoxPrompter } from '../../../shared/ui/inputPrompter'
+import { createWizardTester } from '../../shared/wizards/wizardTestUtils'
+
+class FakeNode extends AWSTreeNodeBase {
+    public constructor(label: string) {
+        super(label)
+    }
+}
+
+describe('searchLogGroup', async function () {
+    const fakeLogNodes: AWSTreeNodeBase[] = []
+    let inputBox: ExposeEmitters<vscode.InputBox, 'onDidAccept' | 'onDidChangeValue' | 'onDidTriggerButton'>
+    let testPrompter: InputBoxPrompter
+
+    before(function () {
+        fakeLogNodes.push(new FakeNode('group-1'), new FakeNode('group-2'), new FakeNode('group-3'))
+        testPrompter = createFilterpatternPrompter()
+
+        inputBox = exposeEmitters(testPrompter.inputBox, ['onDidAccept', 'onDidChangeValue', 'onDidTriggerButton'])
+    })
+
+    it('Wizard accepts inputs', async function () {
+        const testWizard = createWizardTester(new SearchLogGroupWizard(fakeLogNodes))
+        const logGroupSelection = 'group-2'
+        testWizard.logGroup.applyInput(logGroupSelection)
+        testWizard.logGroup.assertValue(logGroupSelection)
```

Comment:
Wizard tests are meant to check the _flow_,  not necessarily the data. These two lines are basically equivalent to:
```ts
const foo: Record<string, any> = {}
foo.groupName = logGroupSelection
assert.strictEqual(foo.groupName, logGroupSelection)
```

Of course, this is probably not obvious without examples of other wizard flows: https://github.com/aws/aws-toolkit-vscode/blob/5dde84bf03bb416e2a74be2084500d31e80771a2/src/test/lambda/wizards/uploadLambdaWizard.test.ts#L28-L47

Notice that the tests basically read like:
* User sees this step first
* User sees this step second
* User never sees this step

Reply:
That makes a lot of sense, it felt redundant to write those. Thanks for providing an example for reference.